test/unit/assorted/imports.test.ts

```diff
@@ -15,7 +15,7 @@ function* walk(root) {
   }
 }
 
-describe.skip('importing mongodb driver', () => {
```

Comment:
@baileympearson I coulda swore I reversed this skip in the previous PR I mistakenly pushed the skip to. I skip it locally because its really slow when I want to rerun the whole suite of unit tests. Anyway, its still passing we should just flip this back.